gateway/mw_oas_validate_request.go

```diff
@@ -60,12 +61,27 @@ func (k *ValidateRequest) ProcessRequest(w http.ResponseWriter, r *http.Request,
 
 	router, err := gorillamux.NewRouter(&oasSpec)
 	if err != nil {
-		return fmt.Errorf("request validation error: %v", err), http.StatusBadRequest
+		return nil, http.StatusOK
 	}
 
 	route, pathParams, err := router.FindRoute(r)
 	if err != nil {
-		return fmt.Errorf("request validation error: %v", err), http.StatusBadRequest
+		return nil, http.StatusOK
+	}
+
+	operation, ok := k.Spec.OAS.GetTykExtension().Middleware.Operations[route.Operation.OperationID]
```

Comment:
Do we need to protect concurrent map access here?

Reply:
or we could use copied OAS object here - https://github.com/TykTechnologies/tyk/pull/4200/files#diff-f5de96d1354ffab65e076f23a58337a7799a212f6e70dea59b56576042ecd69aR57

Reply:
It is already protected in the `Enabled` function of the middleware. We know that map is there.